src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/client/OcculusTabRenderer.java

```diff
@@ -17,6 +17,8 @@
 
 import java.util.List;
 
+import net.minecraft.client.gui.narration.NarratableEntry.NarrationPriority;
```

Comment:
what is this? it's the only change in this file, seems useless, please remove this